app/models/forecast.rb

```diff
@@ -1,15 +1,68 @@
 require 'forecast_io'
+require 'openssl' #evil ssl removing hack for windows since windows doesn't have a root store. I am not going to lie, this is sleezy.
+
+#Hack: http://stackoverflow.com/questions/170956/how-can-i-find-which-operating-system-my-ruby-program-is-running-on Just testing.
+def isWindows?
```

Comment:
You mentioned a way to do this in rake. What did you mean?
Thanks.

On 4/17/2016 1:55 PM, Brandon wrote:
>
> In app/models/forecast.rb 
> <https://github.com/Ditofry/colorado-alarm-api/pull/2#discussion_r59991264>:
>
> > @@ -1,15 +1,68 @@
> >  require 'forecast_io'
> > +require 'openssl' #evil ssl removing hack for windows since windows doesn't have a root store. I am not going to lie, this is sleezy.
> > +
> > +#Hack: http://stackoverflow.com/questions/170956/how-can-i-find-which-operating-system-my-ruby-program-is-running-on Just testing.
> > +def isWindows?
>
> @derekriemer <https://github.com/derekriemer> if we still need this 
> then we could probably just move it to the Rake file and leave it there.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub 
> <https://github.com/Ditofry/colorado-alarm-api/pull/2/files/c829eee94179237ebc2c5228fecd1a6275fa21cb#r59991264>
>

-- 
------------------------------------------------------------------------


    Derek Riemer

  * Department of computer science, third year undergraduate student.
  * Proud user of the NVDA screen reader.
  * Open source enthusiast.
  * Member of Bridge Cu
  * Avid skiier.

Websites:
Honors portfolio <http://derekriemer.com>
Awesome little hand built weather app! 
<http://django.derekriemer.com/weather/>

email me at derek.riemer@colorado.edu <mailto:derek.riemer@colorado.edu>
Phone: (303) 906-2194